src/git/log.js

```diff
@@ -11,6 +11,7 @@ function log(repoPath, done) {
     cwd: repoPath
   }, function(error, stdout, stderr) {
     if (error) {
+      throw new Error(error);
```

Comment:
Also, we could certainly benefit from re-writing the codebase to use (err, cb) style 😸